store/src/java/com/zimbra/cs/account/soap/SoapProvisioning.java

```diff
@@ -3224,4 +3219,10 @@ public void modifyHabGroupSeniority(String habGroupId, String seniorityIndex)
         GetDistributionListMembersResponse resp = invokeJaxb(new GetDistributionListMembersRequest(0, 0, group.getName()));
         return resp.getHABGroupMembers();
     }
+
+    @Override
+    public void resetPassword(Account acct, String newPassword, boolean dryRun) throws ServiceException {
+        // TODO Auto-generated method stub
```

Comment:
I think you have implemented this in the previous comment. 
throw unsupported exception on implement it.